docs/en/ingest-management/fleet/add-fleet-server-kubernetes.asciidoc

```diff
@@ -0,0 +1,392 @@
+====
+
+To use {fleet} for central management, a <<fleet-server,{fleet-server}>> must
+be running and accessible to your hosts.
+
+You can deploy {fleet-server} on Kubernetes and manage it yourself.
+In this deployment model, you are responsible for high-availability,
+fault-tolerance, and lifecycle management of the {fleet-server}.
+
+To deploy a {fleet-server} on Kubernetes and register it into Fleet you will need the following details:
+
+* The *Policy ID* of a Fleet policy configured with the {fleet-server} integration.
+* A *Service token*, used to authenticate Fleet Server with Elasticsearch.
+* For outgoing traffic:
+** The *{es} endpoint URL* where the {fleet-server} should connect to, configured also in the {es} output associated to the policy.
+** When a private or intermediate Certificate Authority (CA) is used to sign the {es} certificate, the *{es} CA file* or the *CA fingerprint*, configured also in the {es} output associated to the policy.
+* For incoming connections:
+** A *TLS/SSL certificate and key* for the {fleet-server} HTTPS endpoint, used to encrypt the traffic from the {agent}s. This certificate has to be valid for the *{fleet-server} Host URL* that {agent}s use when connecting to the {fleet-server}.
+* Extra TLS/SSL certificates and configuration parameters in case of requiring <<mutual-tls,mutual TLS>> (not covered on this document).
+
+This document walks you through the complete setup process, organized into the following sections:
+
+* <<add-fleet-server-kubernetes-compatibility,Compatibility requirements>>
+* <<add-fleet-server-kubernetes-cert-prereq>>
+* <<add-fleet-server-kubernetes-preparations>>
+* <<add-fleet-server-kubernetes-install>>
+
+[discrete]
+[[add-fleet-server-kubernetes-compatibility]]
+= Compatibility
+
+{fleet-server} is compatible with the following Elastic products:
+
+* {stack} 7.13 or later.
+** For version compatibility: {es} >= {fleet-server} >= {agent} (except for
+bugfix releases)
+** {kib} should be on the same minor version as {es}.
+
+[discrete]
+[[add-fleet-server-kubernetes-prereq]]
+= Prerequisites
+
+Before deploying the {fleet-server}, you need to:
+
+* Prepare the SSL/TLS configuration, server certificate, <<fleet-server-hosts-setting>>, and needed Certificate Authorities (CAs).
+* Ensure components have access to the ports needed for communication.
+
+[discrete]
+[[add-fleet-server-kubernetes-cert-prereq]]
+== {fleet-server} and SSL/TLS certificates considerations
+
+This section shows the minimum requirements in terms of Transport Layer Security (TLS) certificates for the {fleet-server}, assuming no mutual TLS (mTLS) is needed. Refer to <<tls-overview>> and <<mutual-tls>> for more information about the configuration needs of both approaches.
+
+There are two main and differentiated traffic flows associated with the {fleet-server}, where each of them has different TLS needs:
```

Comment:
```suggestion
There are two main traffic flows available for {fleet-server}, each with different TLS needs:
```
Just trying to simplify it a bit. :-)

Reply:
What about `There are two main traffic flows related (to or with) {fleet-server}, each with different TLS needs`?

The word `available` for traffic flows doesn't sound accurate to me, as a traffic flow is not something available or unavailable, but more something that exists and happens or doesn't exist. What do you think? :)

Reply:
updated already, let me know your thoughts. I finally wrote:

`There are two main traffic flows for {fleet-server}, each with different TLS requirements:`

Reply:
Looks good. Thanks!